odoo/addons/base/views/res_partner_views.xml

```diff
@@ -371,7 +371,8 @@
                             </aside>
                             <main class="ps-2 ps-md-0">
                                 <div class="mb-1">
-                                    <field name="display_name" class="mb-0 fw-bold fs-5"/>
+                                    <field name="name" class="mb-0 fw-bold fs-5 d-md-none"/>
+                                    <field name="display_name" class="mb-0 fw-bold fs-5 d-none d-sm-block"/>
```

Comment:
Actually, we don't want to show the address in desktop as well. 🙂

Reply:
So maybe we could display the name in all cases? 🤷🏼‍♂️

Reply:
Actually, it should be better to display the complete_name instead of the name I think

Reply:
> Actually, it should be better to display the complete_name instead of the name I think

When we use complete_name, the company name is also displayed with the customer. Should it be shown?

Reply:
Yep, I think it is important to know in which company the partner is.